Use `sum_intlist` and `prod_intlist`

r-barnes opened the pull request:
Differential Revision: D25903430

albanD commented on caffe2/utils/math/reduce.cc:
```diff
@@ -14,12 +8,19 @@
 #include "caffe2/utils/math/elementwise.h"
 #include "caffe2/utils/math/utils.h"
 
+#include <algorithm>
```
Why are all these new imports added if you only add usage for `c10::multiply_integers` ?

r-barnes committed:
Use `sum_integers` and `multiply_integers` (#51146)

Summary: Pull Request resolved: https://github.com/pytorch/pytorch/pull/51146

Differential Revision: D25903430

fbshipit-source-id: 8c5a2bfaeda926c4096fe8f121c1ba71e64ab8db

ngimel reviewed (approved):
I assume it's just codemod, replacing prod_intlist with multiply_integers with the same args?
